src/main/resources/com/cloudbees/plugins/credentials/CredentialsStoreAction/DomainWrapper/newCredentials.jelly

```diff
@@ -63,13 +64,13 @@
                   </f:dropdownListBlock>
                 </j:forEach>
               </f:dropdownList>
-              <!--f:dropdownDescriptorSelector field="credentials" title="${%Kind}" lazy="it"/-->
+              <!--<f:dropdownDescriptorSelector field="credentials" title="${%Kind}" descriptors="${descriptors}"/> -->
```

Comment:
After making these changes I _think_ it works but didn't want to do it in the same PR